### PR TITLE
Fix: icon names work capitalised and lowercase

### LINKED ISSUE #805: [DOCS] Available icons names are misleading
### Description

I found many users has their icon's key value first letter in uppercase, which causing a problem of not showing icons in milestone 

### Screenshots

firefox_Twn30Smhuc
before fixing the issue

firefox_b2glkj4bC8
after fixed

### Additional information

it can be solved either by converting icon value into lowercase on runtime or just by correcting the available icons name on ReadME

### PR DESCRIPTION
**Pull request by nathgoutam93**
## Fixes Issue

Closes #805 

## Check List 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

firefox_Twn30Smhuc

Issue: the milestone component icons are not showing properly because icon names were not matching.

firefox_b2glkj4bC8

Fixed: now icons are showing properly even if the icon names are uppercase

## Note to reviewers

Did some code Refactoring, you can change them if the style do not follows this project 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/807"></a>

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you separate the profile to another PR?

**Comment by nathgoutam93** [comment]
> 

okay, Should i close this one

**Comment by nathgoutam93** [comment]
Done , removed the Profile . I'll make another PR for the Profile.

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you revert the package files?

**Comment by nathgoutam93** [comment]
Wdym by package files? package json files??

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Just revert the changes made in the `package-lock.json` and `package.json` files. That way, the changes made will be reverted. Let us know if you need any help! 😃

File: src/Components/Home/Placeholders.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function Placeholder({ list }) {
     <>
       <Skeleton shape="rectangle" height="3.2rem" className="p-mb-4" />
       <div className="p-d-flex p-flex-wrap p-jc-center">
-        {list.map((user, key) => {
+        {list.map((_, key) => {
```
Comment (emmalearnscode) [inline src/Components/Home/Placeholders.js:11]: Why do you remove user here? Although not used, it makes it clear what in the array is being mapped over and might be used later if further features are added.

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

**Comment by nathgoutam93** [comment]
I'll make another pull request with all the suggestions applied . I'm closing this one cause I'm seeing a problem with the GitHub commit as well as my forked repo giving 500 error code so can't do any new commits

**Pull request closed without merging**